cpp/src/decisiontree/batched-levelalgo/builder_base.cuh

```diff
@@ -507,7 +515,7 @@ struct RegTraits {
    */
   static void computeSplit(Builder<RegTraits<DataT, IdxT>>& b, IdxT col,
                            IdxT batchSize, CRITERION splitType,
-                           cudaStream_t s) {
+                           int& n_large_nodes_in_curr_batch, cudaStream_t s) {
```

Comment:
Why is n_large_nodes_in_curr_batch passed by reference? Is it modified somewhere?

Reply:
it was in my initial version, but you're right, unnecessary here. I have changed it to `const int` 👍🏾